In [1]:
import numpy as np
import pickle
import pandas as pd

In [2]:
# Lets put the values from the first row as follows:

In [3]:
Year = 2014
Present_Price = 5.59
Kms_Driven = 27000
Transmission = 1
Owner = 0
Fuel_Type_cat= 2
Car_Name_cat= 90
Seller_Type_cat= 0


In [4]:
x = np.array([[Year, Present_Price, Kms_Driven, Transmission, Owner, Fuel_Type_cat, Car_Name_cat, Seller_Type_cat]])

In [5]:
x.shape

(1, 8)

In [6]:
model = pickle.load(open('model.pkl', 'rb'))

In [7]:
y_prediction = model.predict(x)

In [8]:
y_prediction

array([[4.0113608]])

## we can see that the ML model prediction is 4.01 lacs, compared to the actual value 3.35 lacs